src/telemetry/metrics.ts

```diff
@@ -38,4 +118,47 @@ export const startMetrics = (serviceName: string): void => {
 
   const meterProvider = new metrics.MeterProvider({ resource, readers });
   api.metrics.setGlobalMeterProvider(meterProvider);
+
+  channel.subscribe(({ fastify }: { fastify: FastifyInstance }) => {
```

Comment:
Should this and below be executed if `METRICS_ENABLED` is not `true`?

Reply:
Added early return instead